tests/integration/golang/aim/experiment/get_experiments_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build integration
+
+package experiment
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetExperimentsTestSuite struct {
+	suite.Suite
+	client   *helpers.HttpClient
+	fixtures *fixtures.ExperimentFixtures
+}
+
+func TestGetExperimentsTestSuite(t *testing.T) {
+	suite.Run(t, new(GetExperimentTestSuite))
+}
+
+func (s *GetExperimentsTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	fixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.fixtures = fixtures
+}
+
+func (s *GetExperimentsTestSuite) Test_Ok() {
+	experiments, err := s.fixtures.CreateExperiments(context.Background(), 5)
+	assert.Nil(s.T(), err)
+	defer func() {
+		assert.Nil(s.T(), s.fixtures.UnloadFixtures())
+	}()
+	var resp []response.GetExperiment
```

Comment:
maybe we can create own type under the hood of `response` package -> type Experiments = []response.GetExperiment ?

Reply:
The main point why I asked that, because `response.GetExperiment` is belongs to `get` endpoint and we know that from the name, but `[]response.GetExperiment` is an array of what? results from `get` endpoint? It is semantically wrong and confusing. So yep, it will still good to fix it and have something like `response.SearchExperiments` or `response.Experiments` or `response.GetExperiments` which give you exact meaning of this struct.

Reply:
@suprjinx you did probably the same which is totally correct.
<img width="673" alt="Screenshot 2023-07-31 at 09 52 49" src="https://github.com/G-Research/fasttrackml/assets/7852635/6ead2390-6ec4-455e-8623-10956eae2722">

Reply:
ok!